backend/de.metas.adempiere.adempiere/client/src/main/java/org/compiere/apps/search/FindHelper.java

```diff
@@ -35,16 +34,13 @@
 import de.metas.adempiere.util.Permutation;
 import de.metas.util.Check;
 
-/**
- * 
- * @author tsa
- * 
- */
+import javax.annotation.Nullable;
```

Comment:
pls avoid changing files while are not related to the issue.
makes reviewing even more harder.

Reply:
what about the boyscout rule?

Reply:
plus: reviewing these side-touched files should be relatively easy, because most of the changes are: 
- nullability annotations
- removing parameter names from comments if they dont have javadoc
- cleanup of final/static when lombok already ads that
- fixing typos

Reply:
boy scout rule says to clear the place near you, not the place in the next forest :)
here, to be more precise, I wasn't sure if there were some changes related to this task or just some cleanup.
thanks for explaining.